packages/fuels/src/cli/commands/deploy/index.ts

```diff
@@ -24,9 +30,9 @@ export async function deploy(config: FuelsConfig) {
 
   for (let i = 0; i < contractsLen; i++) {
     const contractPath = config.contracts[i];
-    const binaryPath = getBinaryPath(contractPath);
-    const abiPath = getABIPath(contractPath);
-    const storageSlotsPath = getStorageSlotsPath(contractPath);
+    const binaryPath = getBinaryPath(contractPath, mode);
+    const abiPath = getABIPath(contractPath, mode);
+    const storageSlotsPath = getStorageSlotsPath(contractPath, mode);
```

Comment:
Maybe for another PR, but it seems to me that we could reuse `getForcProject` for this with maybe a tweak or two to it.